deployment/production-marathon.json

```diff
@@ -22,7 +22,7 @@
       ]
     }
   },
-  "id": "witan-gateway",
+  "id": "OVERRIDDEN_BY_JENKINS",
```

Comment:
I thought this was used in the service route thing?

Reply:
This makes in consistent with how datastore does it.

Reply:
Also the `.`s appear to be re-written by DCOS when creating the internal DNS to hyphens.

Reply:
Ah, ok